ent/encryption/kms_host.cc

```diff
@@ -1004,7 +1004,7 @@ future<encryption::kms_host::impl::key_and_id_type> encryption::kms_host::impl::
      * "GenerateDataKey" API. This creates a new (epiphermal) key, encrypts it 
      * using a named (internal) key, and gives us both raw and encrypted blobs
      * for usage as a local key.
-     * To be able to actually re-use this key again, on decryption of data,
+     * To be able to actually reuse this key again, on decryption of data,
```

Comment:
Note that there is nothing incorrect with "re-use". I would personally consider it a little dated - I would also use the non-hyphenated form "reuse". But I've seen in the past a style guide recommending "re-use" claiming "reuse" is "hideous" :-)